src/oiiotool/oiiotool.cpp

```diff
@@ -53,8 +53,8 @@ using pvt::print_info_options;
 #    define OIIO_UNIT_TESTS 1
 #endif
 
-#ifndef OIIOTOOL_METADATA_HISTORY_DEFAULT
-#    define OIIOTOOL_METADATA_HISTORY_DEFAULT 0
+#ifndef OPENIMAGEIO_METADATA_HISTORYDEFAULT
```

Comment:
Nit / question: Unsure if the intent was to spell this as "...HISTORYDEFAULT" rather than "...HISTORY_DEFAULT"?

Reply:
Not intentional, that appears to be a search-and-replace problem. Thanks so much for spotting.